.github/settings.yml

```diff
@@ -1,12 +1,17 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
 repository:
   name: firefly-tezosconnect
+  description: Firefly Tezos Connect
+  homepage: https://github.com/hyperledger/firefly-tezosconnect
   default_branch: main
-  has_downloads: false
+  has_downloads: true
```

Comment:
What downloads will this repo have?

Reply:
I assume this option is about releases tar files,
which will appear soon when the service release.